docs/user/openstack/customization.md

```diff
@@ -39,6 +41,7 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 * `rootVolume` (optional object): Defines the root volume for instances in the machine pool. The instances use ephemeral disks if not set.
   * `size` (required integer): Size of the root volume in GB.
   * `type` (required string): The volume pool to create the volume from.
+* `zones` (optional list of strings): The names of the availability zones you want to restrict your install to. By default this will contain all zones in your cluster with the status, `available`.
```

Comment:
If unset, the installer will use all available availability zones.

Reply:
Ah that should be removed actually. It does not do that